stdlib/prettyprint/0/prettyprint.rbs

```diff
@@ -0,0 +1,364 @@
+# frozen_string_literal: true
```

Comment:
RBS doesn't understand `frozen_string_literal` directive. ✂️

Reply:
I will remove it